wiki/test-case-started-event-aggregation.md

```diff
@@ -1,7 +1,7 @@
-# TestCaseStartedEvent Aggregation
+# TestCaseStartedEvent aggregation
```

Comment:
Mattias commented earlier that titles should be capitilized, does the same apply here?